### PR TITLE
added new css question

### PR DESCRIPTION
**Pull request by vineet30222**
I added the new question ...that if div is given then how can you fit the image within the div...
It is responsive also....means if size page change then it should automatically fit the image

***new-question-updated-css**
**critical**
**pictorial-explaination**

### CODE REVIEW

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
You have added CSS question but also made changes in the HTML folder.
Remove your file from HTML folder.

File: css/answers/vineet30222-q12.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            HTML | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What are the limits of the text field size?">
```
Comment (dakshkhetan) [inline css/answers/vineet30222-q12.html]: Change this to YOUR question title.

File: css/answers/vineet30222-q12.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+		
+		<img src="../../assets/CN_logo.png" height="72px" alt="Coding Ninjas Logo" />
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">HTML Questions</a>
+                    &gt; What are the limits of the text field size?
```
Comment (dakshkhetan) [inline css/answers/vineet30222-q12.html]: Change this to YOUR question title.

File: html/answers/vineet30222-q12.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
+<html>
```
Comment (dakshkhetan) [inline html/answers/vineet30222-q12.html]: Delete this file.

File: html/index.html [hunk 4]
```diff
@@ -143,6 +143,8 @@ <h4>
         </a>
       </li>
 
+
```
Comment (dakshkhetan) [inline html/index.html]: Remove the extra spaces you have added in this file i.e. undo the changes.

**Comment by vineet30222** [comment]
I have removed the file from the HTML folder...now review the update

**Comment by dakshkhetan** [comment]
Resolve other requested changes too (check my comments corresponding your `css/answers/vineet30222-q12.html` file).
i.e. changing/editing question title at certain places.

**Comment by vineet30222** [comment]
updated the title in the answer file

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
Change/edit question title on line 88 of your `css/answers/vineet30222-q12.html` file.

File: css/answers/vineet30222-q12.html (hunk 2 above)
Comment (vineet30222) [reply css/answers/vineet30222-q12.html]: done!

File: html/index.html (hunk 4 above)
Comment (vineet30222) [reply html/index.html]: done!

**Review by dakshkhetan — APPROVED** [review]

**Pull request merged**